### DISCUSSION TITLE
No module named 'numba.cloudpickle.cloudpickle_fast'

### DISCUSSION THREAD

**Discussion by Nee-DS** [MaartenGr/BERTopic#1848] [Q&A]
I am using Pyinstaller to create an exe for my Bertopic model pipeline. I do not have any issues when I create and execute the exe in my VM GPU. Following is the error I am getting when the pipeline runs in the Azure DevOps pipeline.  Shall I contact Azue, Pyinstaller, or here? Do not know. Please guide 

  File "joblib\numpy_pickle.py", line 648, in load
  File "joblib\numpy_pickle.py", line 577, in _unpickle
  File "pickle.py", line 1212, in load
  File "pickle.py", line 1589, in load_reduce
  File "numba\core\serialize.py", line 110, in _unpickle__CustomPickled
ModuleNotFoundError: No module named 'numba.cloudpickle.cloudpickle_fast'

**Comment by MaartenGr** [comment]
Unfortunately, I'm not familiar with the error. It does seem related to the environment somehow. Perhaps it would help to create a blank environment that did not have numba installed if it did. So installing numba would be completely fresh.

**Comment by Nee-DS** [comment]
I suppose it is from the environment because it is working in the VM. I am now working on the yml file (https://faun.pub/azure-pipelines-and-python-virtual-environments-fdc1089656db) to make sure it gets the same environment as it gets in the VP. I am supposing it is something related to Azurepipeline. Let me know if there is any other thoughts to it.
